xc/xc7/toolchain_wrappers/write_xml_rr_graph

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Please add a description and output help if no arguments are given. IE Make this script like the other `xc/xc7/toolchain_wrappers/write_bitstream` scripts....

Reply:
Unsure this is doable actually, the helper description is given in `xc/xc7/toolchain_wrappers/vpr_common`.

`write_xml_rr_graph` uses the parse_args function, which checks that the correct arguments are passed.

Reply:
parse_args should expect a variable like "HELP" or "DOC" and then disable that if argument match is not found?